docs/sources/alerting/fundamentals/notifications/message-templating.md

```diff
@@ -1,9 +1,9 @@
 ---
 aliases:
   - ../../contact-points/message-templating/
-  - ../../message-templating/
+  - ../../alert-rules/message-templating/
   - ../../unified-alerting/message-templating/
```

Comment:
Another place for comments.

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344